### PR TITLE
Update glossary - remove duplicate entry

### PR DESCRIPTION
**Pull request by thecloudman**
There is a duplicated entry in the glossary which needs to be removed.

### CODE REVIEW

File: specification/glossary.md [hunk 1]
```diff
@@ -88,10 +88,6 @@ The suggested provider-published unit price for a single [Pricing Unit](#pricing
 
 The agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).
 
-<a name="glossary:list-unit-price"><b>Contracted Unit Price</b></a>
```
Comment (cnharris10) [inline specification/glossary.md:91]: Can we also update the entry above with a corrected glossary name and fix any references?

`glossary:list-unit-price` --> `glossary:contracted-unit-price`

**Review by cnharris10 — CHANGES_REQUESTED** [review]

File: specification/glossary.md (hunk 1 above)
Comment (flanakin) [reply specification/glossary.md:91]: +1 I noticed the same and was about to leave a comment 🙂

**Review by flanakin — APPROVED** [review]

File: specification/glossary.md (hunk 1 above)
Comment (udam-f2) [reply specification/glossary.md:91]: Fixed. cc @jpradocueva

Comment (cnharris10) [reply specification/glossary.md:91]: I still see a couple outstanding links to `list-unit-price` ... 
image

**Review by cnharris10 — CHANGES_REQUESTED** [review]

**Review by cnharris10 — APPROVED** [review]

**Comment by jpradocueva** [comment]
This PR was approved by the Members on June 13 call.

**Comment by jpradocueva** [comment]
@udam-f2, @mike-finopsorg could one of you approve this change, which was agreed by the group? Thanks

File: specification/glossary.md [hunk 2]
```diff
@@ -84,11 +84,7 @@ A category of compute resources that can be paused or terminated by the CSP with
 
 The suggested provider-published unit price for a single [Pricing Unit](#pricingunit) of the associated [SKU](#glossary:sku), exclusive of any discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).
 
-<a name="glossary:list-unit-price"><b>Contracted Unit Price</b></a>
-
-The agreed-upon unit price for a single [Pricing Unit](#pricingunit) of the associated SKU, inclusive of negotiated discounts, if present, and exclusive of any other discounts. This price is denominated in the [Billing Currency](#glossary:billingcurrency).
-
-<a name="glossary:list-unit-price"><b>Contracted Unit Price</b></a>
+<a name="glossary:contracted-unit-price"><b>Contracted Unit Price</b></a>
```
Comment (udam-f2) [inline specification/glossary.md:87]: This needs to be moved up to line 55 to keep the alphabetical order.

**Review by udam-f2 — APPROVED** [review]

**Pull request merged**